gulpfile.js

```diff
@@ -213,17 +213,17 @@ gulp.task('build-theme-files', ['styles']);
 
 //watch and run corresponding task on change, process changed files only
 gulp.task('watch', ['build-all'], () => {
-  gulp.watch(withTheme('./static/images/**/*.*'), ['images']);
-  gulp.watch(withTheme('./static/other/**/*.*'), ['other']);
-  gulp.watch(withTheme('./static/styles/**/*.{css,sass,scss}'), ['styles']);
-  gulp.watch(withTheme('./static/fonts/**/*.*'), ['fonts']);
-  gulp.watch(withTheme(nonBaseScripts), ['scripts']);
-  gulp.watch(withTheme(baseScripts), ['base-scripts']);
-  gulp.watch(withTheme('./static/vendor/**/*.{css,sass,scss}'), ['vendor-styles']);
-  gulp.watch(withTheme('./static/vendor/**/*.js'), ['vendor-scripts']);
-  gulp.watch(['./static/vendor/**/*.*', '!./static/vendor/**/*.js',
-    '!./static/vendor/**/*.{css,sass,scss}'
-  ], ['vendor-assets']);
+  let watchOptions = { interval: 1000 };
```

Comment:
please comment why this makes sense..

Reply:
i would suggest to move gulp stuff to an dedicated PR

Reply:
solves gulp high cpu usage:
https://github.com/gulpjs/gulp/issues/634